### DISCUSSION TITLE
Proxy question

### DISCUSSION THREAD

**Discussion by ellielo** [NAIST-SE/AttractingContributors#970] [General]
I am a freelancer and need to use proxy for my work. Please advice me the best proxy service.

**Comment by hernessteve** [comment]
As a freelancer in need of proxy services for your work, your choice of the best proxy service will depend on your specific requirements, such as the nature of your work, your budget, and your need for reliability. Here are a few popular proxy services that cater to different needs:

Luminati: Luminati offers a wide range of proxy types, including residential proxies, mobile proxies, and datacenter proxies. It's known for high quality and reliability. However, it can be relatively expensive.
Smartproxy: Smartproxy provides residential and datacenter proxies with a focus on web scraping and data mining. They offer a user-friendly interface and have competitive pricing.
Oxylabs: Oxylabs offers residential and datacenter proxies with a strong emphasis on web scraping, market research, and e-commerce. They provide a broad range of features and a dedicated account manager for support.

**Comment by MirkosMangle** [comment]
If you're a freelancer looking for the best proxy service, consider to create your own 4G proxy with [iproxy.online](https://iproxy.online/). This way, you have full control over its quality and reliability. You can purchase a dedicated 4G modem and set it up as your proxy server. This option may require a bit more technical knowledge, but it can be a cost-effective and customizable solution. If you prefer a ready-made service, providers like Luminati, Smartproxy, and Oxylabs offer reliable options, depending on your specific needs and budget
